ajax/libs/isomorphic-fetch/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "isomorphic-fetch",
+  "description": "Isomorphic WHATWG Fetch API, for Node & Browserify",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/matthew-andrews/isomorphic-fetch.git"
+  },
+  "author": "Matt Andrews <matt@mattandre.ws> (http://mattandre.ws/)",
+  "license": "MIT",
+  "filename": "fetch-npm-node.js",
+  "keywords": [
+    "fetch",
+    "api",
+    "node",
+    "browserify"
+  ],
+  "npmName": "isomorphic-fetch",
```

Comment:
There are 25 releases on GitHub but 22 on NPM. We should set git auto-update config here.